pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -196,8 +197,17 @@ func (c *Cluster) failMon(monCount, desiredMonCount int, name string) {
 		}
 	} else {
 		// bring up a new mon to replace the unhealthy mon
-		if err := c.failoverMon(name); err != nil {
-			logger.Errorf("failed to failover mon %q. %v", name, err)
+		// only possible if we have enough "ready nodes"
+		if readyNodes, err := k8sutil.GetKubernetesReadyNodes(c.context.Clientset); err != nil {
+			logger.Errorf("list of <Ready> nodes for the mon %q not available: %v", name, err)
```

Comment:
nit: remove 'the'

Reply:
Rook uses periods to separate errors from messages.

```suggestion
			logger.Errorf("list of <Ready> nodes for the mon %q not available. %v", name, err)
```

Reply:
I worry that `not available` might imply there is not an explicit failure here when that is actually the case. I would suggest `failed to get list of <Ready> nodes for mon %q` instead. This follows the `failed to...` pattern present in a lot of Rook's code.